samples/nrf9160/serial_lte_modem/src/slm_at_host.c

```diff
@@ -59,19 +67,40 @@ static size_t at_buf_len;
 static struct k_work cmd_send_work;
 static const char termination[3] = { '\0', '\r', '\n' };
 
+static u8_t uart_rx_buf[UART_RX_LEN][1];
+static char *uart_tx_buf;
```

Comment:
I'd be consistent here and stick to `u8_t`, just as RX buffer.

Reply:
Updated